jenkins-builds/6.0.UP04-17.08-gpu

```diff
@@ -10,6 +10,7 @@
  IPM-2.0.6-CrayIntel-17.08.eb
  IPM-2.0.6-CrayPGI-17.08.eb
  LAMMPS-11Aug2017-CrayGNU-17.08-cuda-8.0.eb         --set-default-module
+ Python-bare/.3.6.2
```

Comment:
You need to write the EasyBuild recipe name with the extension `.eb` and use the flag `--hidden` to hide it, e.g.:
`Python-bare-3.6.2.eb --hidden`